### PR TITLE
Olderzeus NewUI (coverflow w/o animations)

### PR DESCRIPTION
**Pull request by BrianPugh**
This is olderzeus/orzeus's @olderzeus NewUI configurable by the macro `COVERFLOW_UI`.

Some things that should probably still be done/thought about:
1. Update README.md
2. Some stuff in `Makefile` should be in `Makefile.common` and vise-versa.
3. For the files with large changes, i simply changed there to be a `_stock.c` version and a `_coverflow.c` version. For those files I started going through with `#ifdef COVERFLOW_UI` but it quickly became super ugly and hard to follow. For files with few changes, I did the `#ifdef` approach. This naturally leads to some code duplication, but I think its the more maintainable option of the two moving forward.
4. Probably some tweaks to `parse_roms.py`, like not injecting the bmp data if `COVERFLOW_UI` isn't enabled.

So I wouldn't say that this PR is "merge ready", but its 90% there and ready for some feedback.

### CODE REVIEW

**Comment by kbeckmann** [comment]
Really cool! It's a big PR and I will need some time to go through it.

**Comment by BrianPugh** [comment]
it might be easier to compare olderzeus's branch to main rather than this one, and then just look at my last commit which renames some files, which makes the diff look way bigger than it really is.

**Comment by olderzeus** [comment]
Maybe we can create a version inclued no cover show & coverflow horizontal & coverflow vertical style and can change by setting.
but need some time.
